.travis.yml

```diff
@@ -11,6 +11,7 @@ install:
 deploy:
   # dev
   - provider: script
+  #TODO Explore if we can replace installation ID stage uneet-dev with variable 
     script: aws configure set profile.uneet-dev.aws_access_key_id $AWS_812644853088_ID &&
         aws configure set profile.uneet-dev.aws_secret_access_key $AWS_812644853088_SECRET &&
         aws configure set profile.uneet-dev.region ap-southeast-1 &&
```

Comment:
@tuananh1508, we need to replace the Travis variable with the variables we have set in Travis Ci for this repo:
- change `$AWS_812644853088_ID` to `$AWS_ACCOUNT_USER_ID_DEV`
- change `$AWS_812644853088_SECRET` to `$AWS_ACCOUNT_SECRET_DEV`
- change `ap-southeast-1` to `$AWS_DEFAULT_REGION`

Reply:
Done in https://github.com/Unee-T-INS/lambda2sqs/pull/1/commits/e65ada06cb822920e6a1246f9adc8b330ac0c939